testing/cunit_helpers.h

```diff
@@ -0,0 +1,17 @@
+#ifndef CUNIT_HELPERS_H
+#define CUNIT_HELPERS_H
+
+#include <CUnit/Basic.h>
+#include <CUnit/CUnit.h>
+
+#define CU_BUFFER_SIZE 256
+
+#define CU_ASSERT_INT_EQUAL(actual, expected)                                                                          \
```

Comment:
I would avoid adding anything to CUnit's CU namespace. Perhaps this should be OMF_CU_ASSERT_INT_EQUAL?

Reply:
cunit stuff is dropped from this.